src/status_im/utils/utils.cljs

```diff
@@ -82,7 +82,7 @@
   and adds unicode ellipsis in between"
   [address]
   (when address
-    (str (subs address 0 6) "\u2026" (subs address (- (count address) 4) (count address)))))
+    (str (subs address 0 6) "\u2026" (subs address (- (count address) 3) (count address)))))
```

Comment:
what is 3 here? maybe we could put it in a `def` so it is more descriptive?

Reply:
Last 3 characters for the address. Isn't it descriptive enough?

Reply:
yeah I'm not so familiar with cljs functions at times, but seeing this is a substring then yeah you are right, not needed imo :)